src/orion/benchmark/benchmark_client.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.benchmark.client` -- Benchmark client
+================================================================
+
+.. module:: client
+   :platform: Unix
+   :synopsis: Client to use Orion benchmark.
+
+"""
+import datetime
+import importlib
+
+from orion.benchmark import Benchmark, Study
+from orion.core.utils.exceptions import NoConfigurationError
+from orion.storage.base import get_storage, setup_storage
+
+
+def get_or_create_benchmark(name, algorithms=None, targets=None):
```

Comment:
Storage should be configurable like here: https://github.com/Epistimio/orion/blob/develop/src/orion/client/__init__.py#L185

Reply:
yeah, we can. 

I was thinking to make it using global configure and add a python api version interface for such kind configure, just like below. 
```
orion.init(storage=xxx, ...)
```
Because we now have benchmark and experiment client both require similar configure, and there may be more in the future, instead of asking different/same values in each call, we can make user initialize it at the beginning, as for one notebook or py code session, they should share the same configure. And if there is requirement to change some configure in the middle, we can provide additional interfaces.


But for sure, before we have this init interface, let me add the storage configure into benchmark call first. :)

Reply:
Providing the init in the client is a good idea. I would however keep the storage config argument in create_experiment and create_benchmark so that users aren't forced to call init() when unnecessary. It simplifies the process for users. Actually, `setup_storage` kind of plays this role already, we would only need to add it an to the client helper functions.

Reply:
yeah, and even for the config argument, maybe can also make it as a sharable config object (not just storage I think) and used at experiment/benchmark phase.

Reply:
What do you mean by sharable config object? Sharing across what?

Reply:
some thing like this:
orion_config = {“storage": {xxx}, "max_broken": 5, "xxx": "yyy"}

create_benchmark(xxx..., config=orion_config)
create_experiment(xxx..., config=orion_config)

Reply:
Oh ok. Yes. One way would be to set the values in orion.core.config, which is shared. We could make a helper function to help with this.